packages/frontend/src/app/dataContract/[identifier]/DataContract.js

```diff
@@ -56,6 +65,12 @@ function DataContract ({ identifier }) {
 
   useEffect(fetchData, [identifier])
 
+  useEffect(() => {
+    const urlParameters = new URLSearchParams(Array.from(searchParams.entries()))
+    urlParameters.set('tab', tabs[activeTab])
```

Comment:
If i'm not mistaken, page content usually links via `#`, because `?` is query param an basically an argument to the webserver. While we just jumping to the content on the page

Reply:
The "#" sign in the URL is used for linking to content within the page that has a specific id attribute value. The "?" sign is used for passing parameters such as sorting, searching, pagination, and other, including the page number. This is a common approach for pagination, as seen on sites like Etherscan and CoinMarketCap.